packages/slate-editor/src/modules/editor/plugins/withDeserializeHtml/createDeserialize/createTextDeserializer.ts

```diff
@@ -18,7 +19,7 @@ export function deserializeText(extensions: Extension[]) {
         span.appendChild(node.cloneNode(true));
         const { restore } = temporarilyReplaceNode(node, span);
 
-        const result = deserializeHtmlToMarks(extensions)(span, [
```

Comment:
Same here. This single-use calculation was lifted one level up to allow reusing it for a deserialization run.